.github/workflows/ci_cd.yml

```diff
@@ -86,27 +86,20 @@ jobs:
           - [release](https://github.com/pyansys/actions/pulls?q=label%3Arelease+)
 
   code-style:
-    name: Run code style using pre-commit
-    needs: [labeler]
+    name: "Code style"
     runs-on: ubuntu-latest
     steps:
+      - name: "Run PyAnsys code style checks"
+        uses: pyansys/actions/code-style@main
 
-      - name: Install Git and clone project
-        uses: actions/checkout@v3
-
-      - name: Setup Python
```

Comment:
@jorgepiloto Set up (2 words) when used as a verb and setup (1 word) when used as a noun.

Reply:
Oh! Thanks a lot for clarifying this, @PipKat. I will apply it from now on.

Reply:
I applied this suggestion to all the files with this typo.